src/applications/vre/28-8832/wizard/pages/DischargeStatus.jsx

```diff
@@ -27,14 +29,25 @@ const DischargeStatus = ({ setPageState, state = {} }) => {
   };
 
   return (
-    <RadioButtons
-      name="discharge-status"
+    <VaRadio
+      id="dischargeStatus"
+      class="vads-u-margin-y--2"
       label="Have you or your sponsor been discharged form active-duty service in the last year, or do you have less than 6 months until discharge?"
```

Comment:
Found a typo unrelated to your changes. I suppose `form` should be `from`

Reply:
Another good catch, thanks!